### ISSUE TITLE
[README] Standalone binary installation instructions broken

### ISSUE THREAD

**Issue by rhdesmond** [google/osv-scalibr#194]
Following the [README.md instructions](https://github.com/google/osv-scalibr?tab=readme-ov-file#as-a-standalone-binary):
```
$ go install github.com/google/osv-scalibr/binary/scalibr@latest
go: downloading github.com/google/osv-scalibr v0.1.2
go: github.com/google/osv-scalibr/binary/scalibr@latest:
    module github.com/google/osv-scalibr@latest found (v0.1.2), but does not contain package github.com/google/osv-scalibr/binary/scalibr
$ scalibr
Command scalibr not found
```

**Comment by rhdesmond** [comment]
Maybe this is because the directory and binary name are different (used the `osv-scanner` Github project as reference)

**Comment by another-rex** [comment]
I think this is resolved with the 0.1.3 release.

**Comment by rhdesmond** [comment]
Thanks for the quick resolution!
